non-critical-infra/modules/mailserver/mailing-lists.nix

```diff
@@ -8,6 +8,26 @@
   # an encrypted password. Run `nix run .#encrypt-email login -- --help` and
   # follow the instructions.
   mailing-lists = {
+    # nixcon.org
+    "orgateam@nixcon.org" = {
+      forwardTo = [
+        ../../secrets/nixcon-list-email-address.umbriel
```

Comment:
FYI, we don't have to encrypt these. I personally would be inclined to just hard-code "nixcon@nixos.org" here instead.

Reply:
This alias we should keep, it's linked on https://github.com/NixCon (which we can change, but I imagine some people have that stored, and it might be linked elsewhere too)

Reply:
Up until now this is a different recipient. A google group.

Reply:
Replaced with a forward to nixcon@nixos.org and dropped the Google group entirely.